build/charts/antrea/templates/_helpers.tpl

```diff
@@ -9,17 +9,21 @@
 {{- end -}}
 
 {{- define "antreaImageTag" -}}
-{{- if .Values.image.tag }}
-{{- .Values.image.tag -}}
+{{- if .Values.agentImage.tag }}
```

Comment:
agentImage is antrea-agent-ubuntu, shld it be included to controllerimage {{- print .Values.controllerImage.repository ":" (include "antreaImageTag" .) -}}

Reply:
can you please elaborate?, i didn't get your point.

Reply:
with these 3 lines ,  " {{- define "antreaImageTag" -}} {{- if .Values.agentImage.tag }}  {{- .Values.agentImage.tag -}} , it bit confusing here to use same tagname in controller and agent , appending to controller in line no (21-23), can you keep the name generic instead of agentImage in line 12

Reply:
the code is already updated Ref: [comment](https://github.com/antrea-io/antrea/pull/5794#discussion_r1443198069)